ext/pdo_dblib/dblib_stmt.c

```diff
@@ -260,6 +260,101 @@ static int pdo_dblib_stmt_describe(pdo_stmt_t *stmt, int colno)
 	return 1;
 }
 
+static int pdo_dblib_strftime(zval *output, zend_string *format, DBDATEREC *dr)
+{
+	/* mashup of php_date.c php_strftime & FreeTDS convert.c tds_strftime */
```

Comment:
How would you feel about leaning on `ext/date/php_date.c:php_format_date` instead? It's designed to be accessible across extensions. The format strings are different, but it'd provide the same functionality (it's what powers the [date](http://php.net/manual/en/function.date.php) function).

Reply:
That's how I first tried to implement this, but there were a few drawbacks: 
 - milliseconds are only supported when using the DateTime object interface (like in this early [commit](https://github.com/php/php-src/pull/2859/commits/d73e14abb9666f2a877aeea4bbc774049cbf2092))
 - it introduces an unneeded  back and forth conversion to unix timestamp (need to call `php_mktime` before `php_format_date`)